packages/components/back-link/_back-link.scss

```diff
@@ -9,7 +9,8 @@
  */
 
 .nhsuk-back-link {
-  margin-bottom: nhsuk-spacing(3);
+  margin-top: nhsuk-spacing(4);
+  margin-bottom: 0;
```

Comment:
I'd keep margin-bottom - it matches GOV.UK, and it feels safer to generally have margin-bottom on most items.

Reply:
Done in [99222f6](https://github.com/nhsuk/nhsuk-frontend/pull/964/commits/99222f66cd32cd87a791916ac9132f4b0d7ded47). Have updated screenshots in PR description.